mlflow/pyfunc/__init__.py

```diff
@@ -1101,6 +1211,13 @@ def _predict_row_batch(predict_fn, args):
             StringType: np.str_,
         }
 
+        if isinstance(result_type, ArrayType) and isinstance(result_type.elementType, ArrayType):
+            result_values = _convert_array_values(result, result_type)
```

Comment:
This call is missing an arg, right? `spark_primitive_type_to_np_type` is undefined